react/src/components/masthead/components/SprkMastheadMenuIcon/SprkMastheadMenuIcon.js

```diff
@@ -58,33 +58,23 @@ function SprkMastheadMenuIcon({
 
 SprkMastheadMenuIcon.propTypes = {
   /**
-   * Expects a space separated string
-   * of classes to be added to the
-   * component.
+   * A space-separated string of classes to add to the outermost container of the component.
    */
   additionalClasses: PropTypes.string,
   /**
-   * The value supplied will be assigned to the
-   * `data-analytics` attribute on the component.
-   * Intended for an outside
-   * library to capture data.
+   * Assigned to the `data-analytics` attribute serving as a unique selector for outside libraries to capture data.
    */
   analyticsString: PropTypes.string,
   /**
-   * The value supplied will be assigned
-   * to the `data-id` attribute on the
-   * component. This is intended to be
-   * used as a selector for automated
-   * tools. This value should be unique
-   * per page.
+   * Assigned to the `data-id` attribute serving as a unique selector for automated tools.
    */
   idString: PropTypes.string,
   /**
    * If `true`, the Menu will be open on first render.
```

Comment:
"If `true`, the Menu will be open."

I don't think the Menu will ever be open if this is false.